cdn-uploader.js

```diff
@@ -0,0 +1,45 @@
+const ftp = require("basic-ftp")
+
+const config = {
+host: process.env.AKAMAI_CDN_HOSTNAME,
+user: process.env.AKAMAI_CDN_USERNAME,
+password: process.env.AKAMAI_CDN_PASSWORD,
+cpCode: process.env.AKAMAI_CDN_CPCODE,
+port: 21,
+basePath: "./builds"
+}
+
+function UploadFilesFromPath(sources) {
+	process.chdir(config.basePath);
+	console.log("[+] Connecting to CDN...");
+	for (let i = 0; i <= sources.length-1; i++) { 
+		async function Upload() {
+			try {
+				const client = new ftp.Client();
+				await client.connect(config.host, config.port);
+				await client.login(config.user, config.password);
+				await client.useDefaultSettings();
+				console.log(`[+] Starting to upload the content of "${sources[i]}".`);
+				await client.ensureDir(`/${config.cpCode}/${sources[i]}`);
+				await client.clearWorkingDir();
+				await client.uploadDir(sources[i]);
+				client.close()
+				console.log(`[+] Folder "${sources[i]}" uploaded.`);
+			}
+			catch(err) {
```

Comment:
I am not sure if it is a good idea to catch the error, I think that we should abort the upload.

Also, do you think that we could do something like copying into a temporal folder and then rename it at the end? (in order to have a kind of transactional upload)